python/gtsam/tests/test_SfmData.py

```diff
@@ -0,0 +1,79 @@
+"""
+GTSAM Copyright 2010-2019, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+
+See LICENSE for the license information
+
+Unit tests for testing dataset access.
+Author: Frank Dellaert (Python: Sushmita Warrier)
```

Comment:
Ive not seen this kind of separation in author names before. Totally fine if it has been done elsewhere. Otherwise, I think you can list names in the order of the contribution to this file?

Reply:
I think I based this off of John's PR #556 . Let me know if you still want me to change it

Reply:
No, its up to you.